packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -90,6 +96,19 @@ export default function ShareOptionsMenu({
     action: trackAndCopyLink,
   });
 
+  const isMobile = useViewSize(ViewSize.MobileL);
+
+  if (isMobile) {
```

Comment:
Is there anyway we can put such logic rather in contextMenuDrawer or some shared component?
(Same note as on the Modal one)

Reply:
The only option we have is to move this up the chain inside the `PortalMenu` component. I updated this PR to reflect that, let me know what you think